app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -390,6 +400,11 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
         keepQueue = playerData.keepQueue
         timeStamp = playerData.timestamp
 
+        if (PlayerHelper.isPlayFromDownloadEnabled)
```

Comment:
Couldn't we do this directly in `onViewCreated` so that we don't need to create the public `localSearchResult` variable? The variable is only used once, so it's enough to have it locally in the function context.